package.json

```diff
@@ -22,6 +22,7 @@
         "lint": "dtslint types"
     },
     "devDependencies": {
+        "@types/node": "^12.11.2",
```

Comment:
Revert this change to the root `package.json` and just rely on the `reference types` tooling to do the right thing, thanks~ ❤️

Reply:
Done!